DatabaseSeeder/Program.cs

```diff
@@ -0,0 +1,252 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Domain.Entities;
+using Infrastructure.Persistance.Relational;
+using Microsoft.Extensions.Configuration;
+using Infrastructure.Persistance;
+using Microsoft.Extensions.Hosting;
+
+
+namespace DataSeeder
+{
+
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var host = CreateHostBuilder(args).Build();
+            SeedDatabase(host);
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args)
+        {
+
+            var builder = Host.CreateDefaultBuilder(args)
+            .ConfigureServices(services =>
+            {
+                PersistanceConfiguration.ConfigureServices(services, dbtype.DefaultConnection);
+            });
+
+
+            return builder;
+        }
+
+        private static void SeedDatabase(IHost host)
+        {
+            using (var scope = host.Services.CreateScope())
+            {
+                var services = scope.ServiceProvider;
+                var context = services.GetRequiredService<RelationalContext>();
+
+                // Ensure database is created
+                context.Database.EnsureCreated();
+
+                // Seed data
+                if (!context.Cards.Any())
+                {
+                    var cards = GenerateCards();
+                    context.Cards.AddRange(cards); //TODO: Make sure it gets added to the correct table
+                    context.SaveChanges();
+                }
+
+                //Enemy
+                if (!context.Enemies.Any())
+                {
+                    var enemies = GenerateEnemies();
+                    context.Enemies.AddRange(enemies);
+                    context.SaveChanges();
+                }
+            }
+        }
+
+        private static List<Enemy> GenerateEnemies()
+        {
+            var enemies = new List<Enemy>();
```

Comment:
Personligt for overskuelighed, ville jeg sætte hver liste af entities(gælder begge lister, og også kommende lister) der skal indsættes ind i sin egen fil. Det gør det meget mere overskueligt at arbejde med, når man senere hen skal arbejde med dem, eller rette i dem senere hen

Reply:
Du tænker at de skal ligge i hver deres fil. Skulle vi overveje at smide det op på samme måde som med Person.json?